src/main/resources/static/api.yml

```diff
@@ -1055,6 +1085,26 @@ components:
         - probationRegion
         - apArea
         - localAuthorityArea
+    NewPremises:
+      type: object
+      properties:
+        name:
+          type: string
+        apCode:
+          type: string
+          example: NEHOPE1
+        postcode:
+          type: string
+        bedCount:
+          type: integer
+        availableBedsForToday:
+          type: integer
+      required:
+        - name
+        - apCode
+        - postcode
+        - bedCount
+        - availableBedsForToday
```

Comment:
I think that `availableBedsForToday` is a computed value based on the `bedCount` of a premises and actual bookings (https://github.com/ministryofjustice/hmpps-approved-premises-api/blob/2b47d30613509eaf7fc67f07aa800e3ea06fc95a/src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/controller/PremisesController.kt#L70) so I don't think you'll want this in the payload of a the post to create a new premises.

Reply:
Thanks Ed, that makes sense. I'll make the necessary change and push it up shortly